src/openms/include/OpenMS/QC/FragmentMassError.h

```diff
@@ -90,11 +96,36 @@ namespace OpenMS
      */
     void compute(FeatureMap& fmap, const MSExperiment& exp, const QCBase::SpectraMap& map_to_spectrum, ToleranceUnit tolerance_unit = ToleranceUnit::AUTO, double tolerance = 20);
```

Comment:
can this method be const?

Reply:
No, since it stores it's result in a member.